packages/documentation/src/content/docs/integration/deployment/docker-compose.mdx

```diff
@@ -261,9 +353,144 @@ networks:
 volumes:
   pg-data:
   tigerbeetle-data:
-
 ```
 
+<details>
```

Comment:
The complete example seems like it's part of the Networks and volumes subsection. I think it needs a heading. Then the summary can be something like "Click to expand".